src/main/java/com/appium/manager/AppiumParallelMethodTestListener.java

```diff
@@ -35,6 +43,11 @@ public AppiumParallelMethodTestListener() throws Exception {
             prop = ConfigFileManager.getInstance();
             deviceAllocationManager = DeviceAllocationManager.getInstance();
             appiumDriverManager = new AppiumDriverManager();
+            mongoDbUrl = CapabilityManager.getInstance()
+                    .getMongoDbHostAndPort().get("atdHost");
+            mongoDbPort = CapabilityManager.getInstance()
+                    .getMongoDbHostAndPort().get("atdPort");
```

Comment:
`ATDServicePort`

Reply:
https://github.com/saikrishna321/AppiumTestDistribution/blob/4ebdb98f1b6348bc7cd2e4200c5f002943c1bb1b/src/main/java/com/appium/utils/CapabilityManager.java#L74-L75

Have the mapping here